python/ray/experimental/channel/shared_memory_channel.py

```diff
@@ -259,6 +264,15 @@ def __init__(
             self.ensure_registered_as_writer()
             assert self._reader_ref is not None
 
+    def create_reader_ref(self, readers, buffer_size_bytes):
```

Comment:
```suggestion
    def _create_reader_ref(self, readers, buffer_size_bytes):
```

(Use underscore for internal methods)

Can you also add type hints?

Reply:
Done.